documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -306,7 +306,7 @@ The API invoker has to provide in the authorization request (/bc_authorize) a lo
 
 The operator's API exposure platform will:
 
-- Validate user identifier, map it to a telco identifier if applicable, e.g.: map IP to MSISDN. Set the OAuth sub to the unique user id in the operator (Step 2).
+- Validate user identifier, map it to a telco identifier if applicable, e.g.: map IP to MSISDN. Set the OAuth sub to the unique user id in the operator (Step 2). If the operator is not able to identify which user the application wishes to be authenticated by means of the hint provided in the request an `unknown_user_id` error will be returned.
```

Comment:
I actually agree. See my response here https://github.com/camaraproject/IdentityAndConsentManagement/commit/d9bcdbfb315469b7b19fbb71f75c85fa6bfe151b#commitcomment-138318171.

What I mean by "missing" in the PR title is because those error scenarios clarifications are missing in CAMARA documentation with respect to the information that was in the same flows in GSMA. It was explained in the PR description, sorry about that.

>Special notes for reviewers:
>
>This does not add anything "new", it is just to make sure that there is no information documented in GSMA that is missing in CAMARA. Mainly because the GSMA doc is being adapted to reference the CAMARA doc as per https://github.com/camaraproject/IdentityAndConsentManagement/issues/82, and CAMARA must now include all required information.

The included error clarifications are in line with the standard, they were included in the GSMA because in the discussions of the GSMA Opengateway champions it was requested to be included as a clarification in the high level flow description. So it was done and now it is propagated to CAMARA as the only source of truth on what is related to the AuthN/AuthZ flows. 

It is just a documentation matter to fix issue #82, I do not think it will hurt anyone or cause any problems. In the future, when the OIDC profile is ready, we can just include a reference to the CAMARA OIDC profile instead.